packages/shared_preferences/shared_preferences_windows/LICENSE

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
Does flutter/plugins not use an identical license everywhere like flutter/engine?

Reply:
This license is the same as the one in `shared_preferences`, other than the year. Is that what you are referring to?

Reply:
Specifically I was wondering if we are using a set year. flutter/engine does that, to avoid having a ton of license variations (which adds size to Flutter applications, since they include all the licenses).